client/client.go

```diff
@@ -39,8 +105,13 @@ func New(ctx context.Context, apiKey, apiEndpoint, version string, opts ...Clien
 		return resp, err
 	})
 
+	rlClient := &RateLimitedClient{
+		client:      base,
+		rateLimiter: rate.NewLimiter(rate.Every(time.Minute/bucketsPerMinute), rpm/bucketsPerMinute),
```

Comment:
I'm not too familiar with this lib, sorry if it's a silly Q.
Should the first parameter be `rpm/time.Minute` ?

Reply:
absolutely. I was thought I was setting the granularity of the rate limit buckets. I was wrong.